Update/wata_009.txt

```diff
@@ -2686,7 +2686,7 @@ void main()
 //　...結局、ひと気のない休憩コーナーでお喋りをすることになった￥
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　…結局、ひと気のない休憩コーナーでお喋りをすることになった。",
-		   NULL, "...We ended up going to the lounge area, which was vacant, and continuing the conversation.", Line_Normal);
+		   NULL, "...We ended up going to the lounge area, which was vacant, and continued the conversation.", Line_Normal);
```

Comment:
this should be left as present tense consistent with the present tense "going"

"ended up" is a bit of a colloquial thing I guess